File: app/src/main/java/com/xiaomai/geek/GeekApplication.java
package com.xiaomai.geek;

import android.app.Application;
import android.content.Context;

import com.xiaomai.geek.service.InitializeService;

/**
 * Created by xiaomai on 2017/10/25.
 */

public class GeekApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        InitializeService.start(this);
    }

    public static GeekApplication get(Context context) {
        return (GeekApplication) context.getApplicationContext();
    }
}

File: app/src/main/java/com/xiaomai/geek/GeekApplication.kt
package com.xiaomai.geek

import android.app.Application
import com.facebook.stetho.Stetho

/**
 * Created by wangce on 2018/1/26.
 */
class GeekApplication : Application(){
    override fun onCreate() {
        super.onCreate()

        Stetho.initializeWithDefaults(this)
    }
}